addons/website/views/snippets/snippets.xml

```diff
@@ -1206,6 +1206,22 @@
             <we-colorpicker title="Warning" data-customize-website-color="" data-color-type="theme" data-color="warning"/>
             <we-colorpicker title="Error" data-customize-website-color="" data-color-type="theme" data-color="danger"/>
         </we-row>
+        <we-collapse>
+            <we-row string="Grays" class="o_we_gray_preview o_we_toggler">
+                <t t-foreach="9" t-as="i">
+                    <span t-att-title="'Gray '+ str((9-i)*100)"
```

Comment:
For this kind of stuff, use `t-attf-` instead of `t-att-`. It is easier to ready, and without it this won't be translated